tasks/galera.yaml

```
@@ -0,0 +1,58 @@
+---
+- name: Check if grastate.dat exists
+  stat:
+    path: /var/lib/mysql/grastate.dat
+  register: mariadb_grastate
+
+- name: Check if cluster is save to bootstrap
+  when: mariadb_grastate.stat.exists
+  slurp:
+    path: /var/lib/mysql/grastate.dat
+  register: mariadb_bootstrap
+
+- name: Check if mariadb is already runnning
+  stat:
+    path: /var/run/mysqld/mysqld.pid
+  register: mariadb_pid
```

Task is not idempotent but read-only so we should use `changed_when: false`.

```suggestion
  register: mariadb_pid
  changed_when: false
```